database/migration/migrateData.ts

```diff
@@ -3,7 +3,9 @@ import DB from './databaseMethods'
 import { ReviewAssignmentsWithSections } from './types'
 import semverCompare from 'semver/functions/compare'
 import { execSync } from 'child_process'
-import { createIndexes } from './indexes'
```

Comment:
I couldn't find where this was moved to, the file `indexes.ts` is also being removed.

Reply:
Yes, it just uses the single "49_index.sql" script, rather than repeating it in two places.